src/base/traverseFunctions.js

```diff
@@ -146,8 +146,10 @@ export function determineFrustumSet( tile, renderer ) {
 	// at an external tile set.
 	if ( ( stopAtEmptyTiles || ! tile.__contentEmpty ) && ! tile.__externalTileSet ) {
 
-		const error = renderer.calculateError( tile );
-		tile.__error = error;
+		// compute the _error and __distanceFromCamera fields
+		renderer.calculateError( tile );
+
+		const error = tile.__error;
```

Comment:
Ahh! makes sense.. Missed that one! I don't use DEPTH too often, I tend to just use RANDOM_COLOR and gauge the depth/lods based on the layout of the colors.

Reply:
Heh looks like I broke the depth-based debug coloring a bit ago 😬